qt/scientific_interfaces/Inelastic/BayesFitting/BayesFitting.cpp

```diff
@@ -23,8 +25,15 @@ BayesFitting::BayesFitting(QWidget *parent)
   // Connect Poco Notification Observer
   Mantid::Kernel::ConfigService::Instance().addObserver(m_changeObserver);
 
+  auto jobRunner = std::make_unique<MantidQt::API::QtJobRunner>(true);
+  auto algorithmRunner = std::make_unique<MantidQt::API::AlgorithmRunner>(std::move(jobRunner));
+
   // insert each tab into the interface on creation
-  m_bayesTabs.emplace(RES_NORM, new ResNorm(m_uiForm.bayesFittingTabs->widget(RES_NORM)));
+  auto resNormModel = std::make_unique<ResNormModel>();
+  auto resNormWidget = m_uiForm.bayesFittingTabs->widget(RES_NORM);
+  m_bayesTabs.emplace(RES_NORM, new ResNormPresenter(resNormWidget, std::move(algorithmRunner), std::move(resNormModel),
+                                                     new ResNormView(resNormWidget)));
```

Comment:
Ideally we want to hold a unique_ptr to the Presenter. Maybe this isn't possible given the current constraints of the other tabs?